examples/aws-bus/sst.config.ts

```diff
@@ -11,15 +16,12 @@ export default $config({
   async run() {
     const bus = new sst.aws.Bus("Bus");
 
-    const fn = new sst.aws.Function("Fn", {
-      handler: "./src/index.handler",
+    const publisher = new sst.aws.Function("Publisher", {
+      handler: "./src/publisher.handler",
       url: true,
       link: [bus],
     });
-    bus.subscribe("./src/receiver.handler");
 
-    return {
-      url: fn.url,
-    };
+    bus.subscribe("Example", "./src/receiver.handler");
```

Comment:
It would be nice to add this explicit event pattern match to the example.
```
bus.subscribe("Example", "./src/receiver.handler", {
  pattern: { source: ["app.myevent"] }
});
```